src/main/java/org/owasp/html/HtmlEntities.java

```diff
@@ -217,292 +231,2104 @@ public static long decodeEntityAt(String html, int offset, int limit) {
 //  }
 
   /** A trie that maps entity names to codepoints. */
-  public static final Trie ENTITY_TRIE = new Trie(
-      ImmutableMap.<String, Integer>builder()
+  public static final Trie ENTITY_TRIE = new Trie(getEntityNameToCodePointMap());
+
+  private static Map<String, Integer> getEntityNameToCodePointMap() {
```

Comment:
I had to refactor this slightly because the thousands of chained `.put()` calls caused the Java compiler to run out of resources during compilation, due to it tracking the entire stack.